openshift-kube-apiserver/admission/customresourcevalidation/route/validate_route.go

```diff
@@ -45,18 +41,21 @@ func toRoute(uncastObj runtime.Object) (*routev1.Route, field.ErrorList) {
 }
 
 type routeV1 struct {
+	sarc           authorizationv1client.SubjectAccessReviewInterface
+	secretsGetter  corev1client.SecretsGetter
+	validationOpts routecommon.RouteValidationOptions
```

Comment:
If `routecommon.RouteValidationOptions` is a struct then I think you will need a function.

Reply:
Addressed